packages/next/src/server/node-environment-extensions/error-inspect.tsx

```diff
@@ -0,0 +1,3 @@
+import { patchErrorInspect } from '../patch-error-inspect'
+
+patchErrorInspect()
```

Comment:
Seemingly unnecessary indirection but I went back and forth when to match and if the patch needs arguments. Keeping the indirection for development for now.